### ISSUE TITLE
OpenSourceTables: size(...) in external C function

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1206]
**Reported by beutlich on 11 Jul 2013 11:55 UTC**
Should
```
void* ModelicaStandardTables_CombiTimeTable_init(const char* tableName,
                                                 const char* fileName,
                                                 double* table, int nRow,
                                                 int nColumn,
                                                 double startTime,
                                                 int* columns,
                                                 int nCols, int smoothness,
                                                 int extrapolation);
```
be changed to
```
void* ModelicaStandardTables_CombiTimeTable_init(const char* tableName,
                                                 const char* fileName,
                                                 double* table, size_t nRow,
                                                 size_t nColumn,
                                                 double startTime,
                                                 int* columns,
                                                 size_t nCols, int smoothness,
                                                 int extrapolation);
```
?

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1206

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 11 Jul 2013 12:10 UTC**
Yes nRow and nColumn must have type size_t

**Comment by modelica-trac-importer** [comment]
**Comment by beutlich on 11 Jul 2013 12:51 UTC**
Resolved by 1a914863a6c99690ec1956d3da7e3406c1c34164.
